<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_fixo</class>
 <widget class="QMainWindow" name="Tela_fixo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_fixos">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>470</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>470</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Fixos</string>
    </property>
    <widget class="QLabel" name="label_largura">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Largura</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_largura">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_altura">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Altura</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_altura">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>40</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_espessura">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="placeholderText">
      <string>Espessura</string>
     </property>
     <item>
      <property name="text">
       <string>08mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10mm</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_quantidade">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Quant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_quantidade">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_calcular">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancelar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_margem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_margem">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Margem</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_unitario">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>9</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Total Unit√°rio:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_geral">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>105</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Total Geral:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_vidro">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>116</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="placeholderText">
      <string>Vidro</string>
     </property>
     <item>
      <property name="text">
       <string>Temperado Incolor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Temperado Fume</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Temperado Verde</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_cifra_unitario">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ff0000;&quot;&gt;R$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cifra_geral">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>145</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ff0000;&quot;&gt;R$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_valor_total_unitario">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: red;
	font-size: 22px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>0000,00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_valor_total_geral">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>153</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: red;
	font-size: 22px;
	font-weight: bold;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>0000,00</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_largura</tabstop>
  <tabstop>lineEdit_altura</tabstop>
  <tabstop>lineEdit_quantidade</tabstop>
  <tabstop>comboBox_espessura</tabstop>
  <tabstop>comboBox_vidro</tabstop>
  <tabstop>lineEdit_margem</tabstop>
  <tabstop>pushButton_calcular</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
